src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,43 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_NUMBER = 10;
+    private final K[] keys;
+    private final V[] values;
+    private int size;
+
+    public StorageImpl() {
+        this.keys = (K[]) new Object[MAX_ARRAY_NUMBER];
+        this.values = (V[]) new Object[MAX_ARRAY_NUMBER];
```

Comment:
```suggestion
        keys = (K[]) new Object[MAX_ARRAY_NUMBER];
        values = (V[]) new Object[MAX_ARRAY_NUMBER];
```
`this` - is redundant